### PR TITLE
changes for ai_copilot

### PR DESCRIPTION
**Pull request by Nikhil7174**

### CODE REVIEW

File: components/ping_ai_copilot/extension/content/content_scripts/content.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import "../ui/style.css"
```
Comment (udaybansal19) [inline components/ping_ai_copilot/extension/content/content_scripts/content.js]: Is this import working? this is not a module to imported in a js file and might require to use styled components or to declare this as a static resource

File: components/ping_ai_copilot/extension/content/content_scripts/content.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+const initializeExtension = () => {
+  document.removeEventListener('mouseup', handleTextSelection);
+  document.addEventListener('mouseup', handleTextSelection);
+  document.addEventListener('click', handleDocumentClick);
+}
+
+const handleTextSelection = (event) => {
+  const selection = window.getSelection();
+  const selectedText = selection.toString().trim();
```
Comment (udaybansal19) [inline components/ping_ai_copilot/extension/content/content_scripts/content.js:13]: why do we need to trim this?

File: components/ping_ai_copilot/extension/content/content_scripts/content.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  }
+}
+
+const sendTextToSummarize = (text) => {
+  showSummaryBox(true);
+  try {
+    chrome.runtime.sendMessage({ action: 'summarize', text: text });
+  } catch (error) {
+    console.error('Failed to send message:', error);
+    showSummaryBox(false, 'An error occurred. Please refresh the page and try again.');
```
Comment (udaybansal19) [inline components/ping_ai_copilot/extension/content/content_scripts/content.js:72]: will the user need to refresh. or user can click somewhere else or select again to trigger this again?

**Review by udaybansal19 — DISMISSED** [review]
lgtm. left some doubts I had as comments

File: components/ping_ai_copilot/extension/content/content_scripts/content.js (hunk 1 above)
Comment (Nikhil7174) [reply components/ping_ai_copilot/extension/content/content_scripts/content.js]: there is no need to import css in content scripts, its css file path is added in manifest itself. will remove this import

**Review by udaybansal19 — APPROVED** [review]

**Pull request merged**